src/lib/genesis_constants/genesis_constants.ml

```diff
@@ -223,6 +223,8 @@ module T = struct
     ; max_action_elements : int
     ; zkapp_cmd_limit_hardcap : int
     ; minimum_user_command_fee : Currency.Fee.Stable.Latest.t
+    ; sync_ledger_default_subtree_depth : int
```

Comment:
These really shouldn't be a part of `Genesis_constants`.

Reply:
I wasn't sure where it should be. It seems like we are working towards removing compile config. Any suggestion?

Reply:
Maybe create something new like `Daemon_constants` that should include a lot of what's in `Genesis_constants`?

Reply:
There are two constants (three if we extracted `account_subtree_height` later) and they need to be threaded through a few different modules- bootstrap_controller, block_producer and consensus. However, they are used only in syncable_ledger.ml and so exposing it along the way doesn’t seem nice in terms of readability. Grouping and passing them in contexts seemed better.

Reply:
Moved it back to compile time. Will propose this in a different PR